docs/devops/configuration.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Configuration
+hide_title: true
+sidebar_position: 1
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Configuration
+## Basics
+### accounts
+Account is a list that includes the addresses and amount of tokens initially allocated. The value of each node has must be consistent.
+| Value Type| Default Value| Unit | Optional |
```

Comment:
The reason why the markdown table is not rendering is that there is no blank line before the table, causing it to be recognized as part of the previous paragraph. Adding a blank line after "12L" will display it correctly. Other tables in the current file have the same issue.